roles/ota/apps/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Clone repository
+  ansible.builtin.git:
+    repo: "{{ ota_source_repository }}"
+    dest: /home/{{ ansible_user }}/{{ ota_directory }}
+    version: "{{ ota_source_repository_branch }}"
+    force: true
+    key_file: /home/{{ ansible_user }}/.ssh/ota-github-bot-key
```

Comment:
For me, this doesn't change anything since the key name in the managed directory is also hardcoded. To avoid conflicts, the name is prefixed with `ota`, making collisions unlikely to occur.